src/storage/src/mem_table.rs

```diff
@@ -582,7 +574,7 @@ impl<S: StateStoreWrite + StateStoreRead> LocalStateStore for MemtableLocalState
         }
         self.inner.ingest_batch(
             kv_pairs,
-            delete_ranges,
+            vec![],
```

Comment:
Can we remove `delete_ranges` in `StateStoreWrite::ingest_batch` as well?

Reply:
Temporarily we cannot, because in memory and sled state store, it depends on passing range delete to remove the keys below table watermark.